interactions/api/models/gw.py

```diff
@@ -258,10 +258,14 @@ class GuildJoinRequest(DictSerializerMixin):
 
     :ivar Snowflake user_id: The user ID of the event.
     :ivar Snowflake guild_id: The guild ID of the event.
+    :ivar Optional[Any] request: The actual request representing the event. This pertains primarily to _CREATE, but maybe _UPDATE as well.
+    :ivar Optional[str] status: The status of the event, which pertains to _CREATE.
     """
 
     user_id: Snowflake = field(converter=Snowflake)
     guild_id: Snowflake = field(converter=Snowflake)
+    request: Optional[Any] = field(default=None)
```

Comment:
There are no info about `request` to properly annotate it?

Reply:
The only thing I've gotten from my last testing (months ago) was a dictionary, however I don't have the logs for it nor is it documented in the docs (nor can I test it further at the moment) nor if I know if it's still there now compared to local testing.

Reply:
lol. Okay then